lib/mail_room/cli.rb

```diff
@@ -42,10 +42,12 @@ def initialize(args)
     # Start the coordinator running, sets up signal traps
     def start
       Signal.trap(:INT) do
+        structured_logger.info(action: "Interrupt signal received")
         coordinator.running = false
```

Comment:
The coordinator should be able to trigger each of the configured watchers+mailboxes to indicate that they are stopping in logs.

Reply:
Great, I've added it to the `quit` method in `mailbox_watcher`.